### ISSUE TITLE
Switching from Less to Sass

### ISSUE THREAD

**Issue by ryanwellsdotcom** [jaxcore/sails-react-redux-boilerplate#5]
I'm trying to convert the project from less to sass, but I'm running into snags. It looks like the documentation may be out of date.

**Comment by dsteinman** [comment]
Yes, this boilerplate is out of date and unfortunately I'm not even using it anymore. I'm pretty sure Less came out of the box with Sailjs, so you'd have to go through the docs at sailsjs.org to figure out how to switch to Sass.
